browser/brave_browser_main_parts_linux.cc

```diff
@@ -0,0 +1,12 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/brave_browser_main_parts_linux.h"
+#include "brave/browser/browsing_data/brave_clear_browsing_data.h"
+
+void BraveBrowserMainPartsLinux::PreShutdown() {
```

Comment:
I thought I left a comment earlier that we don't need linux/mac/etc.. we only need BraveBrowserMainParts::PreShutdown